tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -516,3 +534,34 @@ Check And Uninstall Operator In Openshift
         END
     END
     Close All Browsers
+
+Validate URls
+    [Documentation]   validates  urls
+    [Arguments]  ${urls}
+    FOR  ${url}   IN   ${urls}
+        Log To Console  ${url}
+        ${status}=    Check HTTP Status Code    link_to_check=${url}
+        Log To Console     ${url}  gets status code ${status}
+    END
+
+Get Urls and validation of quickStrat Tiles
+    [Documentation]     Clicks on the side window and  validate all Url
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
```

Comment:
could you make it more self-explanatory?

You could look at `Verify Quick Starts Work As Expected When All Steps Are Marked As Yes` for an example, which  IMO it's a easier to be read